Cargo.toml

```diff
@@ -35,11 +35,11 @@ dotenv = "0.15.0"
 # In order to use dojo-test-utils, we need to explicitly declare the same patches as them in our Cargo.toml
 # Otherwise, underlying dependencies of dojo will not be patched and we will get a compilation error
 # see https://github.com/dojoengine/dojo/issues/563
-dojo-test-utils = { git = 'https://github.com/dojoengine/dojo', rev = "24e8a78" }
+dojo-test-utils = { git = 'https://github.com/jobez/dojo', branch = "make_dojo_test_utils_build_optional", features = ["skip-build"] }
```

Comment:
will update to just point to dojo once this is merged: https://github.com/dojoengine/dojo/pull/634